README.adoc

```diff
@@ -30,17 +29,21 @@ The zip-compressed archives can be downloaded from https://pi4j-download.com[pi4
 === CrowPi OS
 
 * support for https://www.elecrow.com/crowpi-compact-raspberry-pi-educational-kit.html[CrowPi]
+* link:https://pi4j-download.com/crowpi-main.img.zip[Download Latest CrowPi Image]
```

Comment:
```suggestion
* link:https://pi4j-download.com/latest.php?flavor=crowpi[Download Latest CrowPi Image]
```

This helper script will always point to the latest release according to semver, e.g. `2.1.1 > 2.1.0 > 2.0.0 > 1.5.4 > 1.5.0 > 1.0.0 > main`. This link has also been used in the updated docs PR and the dynamic Pi4J README uses the same logic.